File: Iniciante/1013 - O Maior/JavaScript/1013.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split(" ");
  const [a, b, c] = lines.map(Number);
  const higherValue = findHigherValue(a, b, c);
  console.log(`${higherValue} eh o maior`);
}

function findHigherValue(a, b, c) {
  return Math.max(a, b, c);
}

main();

File: Iniciante/1013 - O Maior/Python/1013.py
def main():
    a, b, c = map(int, input().split())
    higher_value = find_higher_value(a, b, c)
    print(f"{higher_value} eh o maior")


def find_higher_value(a, b, c):
    return max(a, b, c)


if __name__ == "__main__":
    main()